packages/web-components/fast-foundation/src/combobox/combobox.ts

```diff
@@ -673,6 +679,15 @@ export class FASTCombobox extends FormAssociatedCombobox {
         }
     }
 
+    /**
+     * Resets the value to the initial value.
+     *
+     * @internal
+     */
+    private resetValue() {
+        this.value = this.initialValue ?? "";
+    }
+
```

Comment:
Just a note - this probably would've been a good separate PR for the sake of tracking, transparency, folks finding relevant fixes via search in the future, etc :)